.gitignore

```diff
@@ -112,6 +112,9 @@ ehthumbs_vista.db
 # Folder config file
 Desktop.ini
 
+# VIM tempfiles
+*.swp
```

Comment:
In that case we probably need `*.swo` as well :)

Reply:
I'll update this PR with *.swo and *~ as well ... i never have more than one swap file, so I wasn't aware of .swo :)